### PR TITLE
Parallax Component.

### PR DESCRIPTION
**Pull request by jsbranco**
The following pull request is to add the Parallax component into react-materialize.

This is my first contribution to this repository, hence please let me know if it matches with the expectations or if there is anything I should correct in order to have this contribution accepted.

### CODE REVIEW

File: src/Parallax.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          <img src={imageSrc}/>
+        </div>
+      </div>
+    );
+  }
+}
+
+Parallax.propTypes = {
+  className: PropTypes.string,
+  imageSrc: PropTypes.string,
```
Comment (isaiah) [inline src/Parallax.js]: Please add some description, this will be shown in the document site.

File: src/Parallax.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { Component, PropTypes } from 'react';
+import cx from 'classnames';
+
+class Parallax extends Component {
+  componentDidMount () {
+    const {
+      className,
+    } = this.props;
```
Comment (alextrastero) [inline src/Parallax.js]: this const isn't used right?

File: src/Parallax.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    $('.parallax').parallax();
+  }
+
+  render () {
+    const { className, imageSrc } = this.props;
+    return (
+      <div className={cx('parallax-container', className)} >
```
Comment (alextrastero) [inline src/Parallax.js]: extra whitespace at end

File: src/Parallax.js (hunk 1 above)
Comment (alextrastero) [inline src/Parallax.js]: No comma dangle :)

**Comment by jsbranco** [comment]
Hi, thank you both @isaiah  and @alextrastero for the feedback.   I have committed the changes as suggested.

Regards
Junio

**Comment by alextrastero** [comment]
Thanks @jsbranco

**Comment by isaiah** [comment]
@jsbranco Thanks for the fix, could you squash your commits? I'll merge after that.

**Comment by jsbranco** [comment]
@isaiah  Sorry since this is the first time i am doing a PR.. by squash you mean a merge of the branches? Not entirely sure if that's what you mean.

**Comment by alextrastero** [comment]
@isaiah @jsbranco there's no real need to squash or merge manually as github has a button that will do that for you :)

**Comment by alextrastero** [comment]
But what @isaiah meant was this: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by isaiah** [comment]
@alextrastero good to know

**Comment by jsbranco** [comment]
thanks @alextrastero and @isaiah   much appreciated!

**Pull request merged**